### PR TITLE
Relaunch - Clarify old site as live until..

### LINKED ISSUE #5443: Relaunch doc page, unclear what happens after step 3
Re: https://pantheon.io/docs/relaunch/   see  **Relaunch Procedure**

Priority: Low

## Issue Description

It is unclear to me what happens in  **Relaunch Procedure**  after completing Step 2 and 3.  Is the old site still live at that point?

## Suggested Resolution

Assuming the old site is still live at that point, and the new site is not live until Step 5, add this wording after Step 3:

"The old site will still be live at this point.  The new site does not become live until Step 5. "

### PR DESCRIPTION
**Pull request by alexfornuto**
DNS is updated.

Closes #5443

## Effect
PR includes the following changes:
- Clarifies confusion around relaunch procedure brought up by @rick02840
- Also caught some outdated DNS info (see #5428)

## Remaining Work
- [x] :+1: from @rick02840 
- [x] Copy Review

## Post Launch
**Do not remove** - To be completed by the docs team upon merge:
- [ ] Update Status Report
- [ ] Remove from the [project board](https://github.com/pantheon-systems/documentation/projects/14)

### CODE REVIEW

File: source/content/relaunch.md [hunk 1]
```diff
@@ -77,6 +78,8 @@ For a fast, smooth relaunch, consider having two browser tabs open, one with the
 
   Do **not** update DNS yet. The Site Dashboard instructs you to configure DNS after connecting the domain, however for this relaunch procedure you should not change record values until instructed in step 5 below.
 
+  Until you update the DNS records (and they propogate), traffic will still be directed to the old site.
```
Comment (alexfornuto) [inline source/content/relaunch.md]: This is the only line related to the issue, the rest is me scope-creeping like a mutha.

**Comment by pantheondocs** [comment]
Lighthouse Audit Results:

| Description | Live | pr-5447 | Difference |
| --- | --- | --- | --- |
| Performance Score | `81` | `62` | `-19` |
| Best Practice Score | `93` | `86` | `-7` |
| SEO Score | `96` | `96` | `0` |
| Accessibility Score | `96` | `93` | `-3` |

Lighthouse Audit Reports:

- [`pr-5447` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/17806/artifacts/0//lighthouse_data/lighthouse-audit-dev.html)
- [`Pantheon Docs` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/17806/artifacts/0//lighthouse_data/lighthouse-audit-reference.html)

**Comment by pantheondocs** [comment]
Lighthouse Audit Results:

| Description | Live | pr-5447 | Difference |
| --- | --- | --- | --- |
| Performance Score | `70` | `69` | `-1` |
| Best Practice Score | `86` | `86` | `0` |
| SEO Score | `96` | `96` | `0` |
| Accessibility Score | `93` | `93` | `0` |

Lighthouse Audit Reports:

- [`pr-5447` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/17807/artifacts/0//lighthouse_data/lighthouse-audit-dev.html)
- [`Pantheon Docs` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/17807/artifacts/0//lighthouse_data/lighthouse-audit-reference.html)

**Comment by rick02840** [comment]
I am following up to this to explain what happened when we went live on Thursday, in case some other changes are needed to the docs and/or how the procedure works on Pantheon.  

I submitted a ticket on this, and below is copy / pasted from that. 

Thanks,

Rick 

=====================

Hi,

I am following up on this just to let you know what happened when we went live on Thursday.

Domain: https://cardozo.yu.edu/
Old site: https://live-yucardozo.pantheonsite.io/ (Drupal 7)
New site: https://live-cardozo.pantheonsite.io/ (Drupal 8)

Reference this: https://pantheon.io/docs/relaunch#relaunch-procedure

**7:45 AM**

I did Steps 2 and 3 (removed domain from old site and added it to new site). I did this 1 hour 15 minutes early to make sure HTTPS was provisioned before Step 5 happened.

At this point, old site was still live and HTTPS was still working, all is good.

**7:50 AM**

I noticed that the new site was now live, HTTPS was working, so all seemed good — but I was a bit confused as to why the new site was already appearing when DNS had not switched.

**8:30 AM**

I started to get reports that HTTPS was not working, it may have started before then. The new site was still the one that was live.

I think this may have occurred when HTTPS became provisioned for the new site.

**9:00 AM**

DNS switch was done per pre-arranged time with IT folks — at that point all was good.

Note that we had pre-arranged a DNS change time with the folks who handled DNS (9:00AM), so we did not have control to be able to change DNS when we wanted to.

**So, for roughly 30 minutes, possibly more, users were getting an SSL error when going to the new site.**

This is perhaps an unusual case because of the pre-arranged time for DNS change, otherwise we could have done the DNS change as soon as we saw that HTTPS was provisioned. But I thought I would relay this info in case there is something that could be done about this in the future.

Thanks,

Rick Hood, Lead Drupal Developer
Common Media // 278 Main St. Suite 205, Greenfield MA 01301
commonmedia.com // 413 259 6977

**Comment by dficker** [comment]
@rick02840 Thanks for your comment above confirming that Step 3 makes the site live. (Looks like maybe there was a few minutes where this was not the case, but within 5 minutes it was.) I just noticed this change and thought it was incorrect.

For the relaunch process to work, it does need to be done in pretty much real-time succession. If you needed to schedule the DNS an hour or two ahead, you could have sent over the DNS info to be done ASAP and that may have made the downtime as short as possible.

To describe what happens, when you do Step 2, that starts the old site's certificate to be deleted in an hour or so. When you do Step 3, that makes the new site live, but also starts the new site's certificate to be created. If you are able to change DNS shortly after the new site has a valid certificate and your DNS TTL is low, as the Relaunch process describes, then the relaunch can be done with no downtime.

I'm sorry this was unclear and we'll work to make it more clear in the future.

**Pull request merged**